flybywiresim/aircraft
# fix(380/fuel): reset valvestates when refueling

Fixes #[issue_no]

## Summary of Changes

This PR tries to resolve an issue where after landing the fuel system is in a state where it cannot be refueled properly as the valves aren't in the correct position.

The current solution is to simply reset the valves to their default position and then have the fuel system recalculate the way they should be triggered.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):\_chaoz_

## Testing instructions


1. Spawn c/d
2. load some fuel for a flight, in either fast or real
3. monitor fuel to see if it loads what is requested
4. check no fuel gets transfered after fueling ends
5. takeoff
6. check that fuel transfers are happening if required
7. monitor fuel until landing to make sure nothing weird happens
8. land
9. after landing and parking up try loading fuel again in either fast or real speed
10. monitor fuel to make sure it all happens as expected

Bonuspoints: Repeat test but spawn on the runway instead


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
